circle.yml

```diff
@@ -72,6 +75,9 @@ test:
         # Try to build production JS
         - yarn run build-prod
 
+        # Check for vulnerable dependencies
+        - ./vendor/bin/security-checker security:check
```

Comment:
👍 
Should we remove the same check in the `build.sh` script then ?
https://github.com/EnMarche/en-marche.fr/blob/master/scripts/build.sh#L5
I don't see the point of checking it during the build 🤔

Reply:
Done.